deploy/operator.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: pgo-deployer-sa
-  namespace: pgo
```

Comment:
I really doubt about it

Reply:
Could you please explain why ? It is `ServiceAccount ` and we have namespace for 'ClusterRoleBinding'.

Reply:
I thought kubectl is forbidding service-account creation without namespace specified